src/pages/index.tsx

```diff
@@ -1,62 +1,142 @@
-import Head from 'next/head';
-import Image from 'next/image';
 
-const Home: NextPage = () => {
+import { Gopher, HomeHero, HomeSection, QuickLinks } from '../components/UI/homepage';
+
+import {
+  CONTRIBUTING_PAGE,
+  DOCS_PAGE,
+  ETHEREUM_FOUNDATION_URL,
+  ETHEREUM_ORG_RUN_A_NODE_URL,
+  ETHEREUM_ORG_URL,
+  GETH_REPO_URL
+} from '../constants';
+
+const HomePage: NextPage = ({}) => {
   return (
-    <div>
-      <Head>
-        <title>Create Next App</title>
-        <meta name='description' content='Generated by create next app' />
-        <link rel='icon' href='/favicon.ico' />
-      </Head>
+    <>
+      {/* TODO: add PageMetadata */}
 
       <main>
-        <h1>
-          Welcome to <a href='https://nextjs.org'>Next.js!</a>
-        </h1>
-
-        <p>
-          Get started by editing <code>pages/index.tsx</code>
-        </p>
-
-        <div>
-          <a href='https://nextjs.org/docs'>
-            <h2>Documentation &rarr;</h2>
-            <p>Find in-depth information about Next.js features and API.</p>
-          </a>
-
-          <a href='https://nextjs.org/learn'>
-            <h2>Learn &rarr;</h2>
-            <p>Learn about Next.js in an interactive course with quizzes!</p>
-          </a>
-
-          <a href='https://github.com/vercel/next.js/tree/canary/examples'>
-            <h2>Examples &rarr;</h2>
-            <p>Discover and deploy boilerplate example Next.js projects.</p>
-          </a>
-
-          <a href='https://vercel.com/new?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app'>
-            <h2>Deploy &rarr;</h2>
-            <p>Instantly deploy your Next.js site to a public URL with Vercel.</p>
-          </a>
-        </div>
-      </main>
+        <Stack spacing={4}>
+          <HomeHero />
+
+          {/* SECTION: What is Geth */}
+          <HomeSection
+            imgSrc='/images/pages/gopher-home-front.svg'
+            imgAltText='Gopher greeting'
+            sectionTitle='What is Geth'
+            linkLabel='Get started with Geth'
+            buttonHref={`${DOCS_PAGE}/getting-started`}
+          >
+            <Text fontFamily='"Inter", sans-serif' lineHeight='26px'>
+              Geth (go-ethereum) is a{' '}
+              <Link href='https://go.dev/' isExternal variant='light'>
+                Go
+              </Link>{' '}
+              implementation of{' '}
+              <Link href={ETHEREUM_ORG_URL} isExternal variant='light'>
+                Ethereum
+              </Link>{' '}
+              - a gateway into the decentralized web.
+            </Text>
+
+            <Text fontFamily='"Inter", sans-serif' lineHeight='26px'>
+              Geth has been a core part of Etheruem since the very beginning. Geth was one of the
+              original Ethereum implementations making it the most battle-hardened and tested
+              client.
+            </Text>
+
+            <Text fontFamily='"Inter", sans-serif' lineHeight='26px'>
```

Comment:
Made an issue to clean up duplicate styles after we merge the downloads and homepage. Not a blocker for this.

https://github.com/ethereum/geth-website/issues/15